src/pdf/DayOverview/DayOverviewPdfButton.tsx

```diff
@@ -64,9 +66,29 @@ export interface DayOverviewPdfData {
     parcels: ParcelForDayOverview[];
 }
 
-type DayOverviewPdfErrorType = ParcelForDayOverviewErrorType;
+type DayOverviewPdfErrorType = ParcelForDayOverviewErrorType | CongestionChargeError;
 export type DayOverviewPdfError = { type: DayOverviewPdfErrorType; logId: string };
 
+const addCongestionChargeDetailsForDayOverview = async (
+    parcels: ParcelForDayOverview[]
+): Promise<ParcelForDayOverviewResponse> => {
+    const postcodes = parcels.map((parcel) => parcel.client?.address_postcode || null);
+
```

Comment:
We need it to return null so that we get a false back for that parcel
Otherwise, for example, we may only get back an object with a length of 3 back from the congestion charge check, even though we are checking 4 parcels, and so the parcels and congestion charge data won't line up

Reply:
Thats great, thanks

Reply:
Good point! The comments above were from when I was using a for loop instead of map so probably don't make total sense now. 
"get a false back" was in reference to when it is passed to the congestion charge edge function (where null will return { postcode: null, congestionCharge: false }. 
But it's true that removing `|| null` with map would get basically the same result, with only an added possibility of undefined, done :)